main.py

```diff
@@ -2,6 +1,44 @@
+from db.models import Genre, Actor
 
-from django.db.models import QuerySet
 
+def main() -> None:
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
 
-def main() -> QuerySet:
-    pass
+    Actor.objects.create(first_name="George", last_name="Klooney")
```

Comment:
There is a typo in the actor's name 'George Klooney'. It should be 'George Clooney'. This is corrected later in the code, but it's better to use the correct spelling initially.